addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/UrlUtils.java

```diff
@@ -139,4 +142,95 @@ public static List<Map<String, String>> confirmUsableParameters(
         }
         return paramGroups;
     }
+
+    /**
+     * Removes all html tags from a given content string.
+     *
+     * @param str the content string to be stripped.
+     * @return a string without html tags.
+     */
+    public static String removeTags(String str) {
+        return str.replaceAll("(?s)<.*?>", "");
+    }
```

Comment:
For consistency and to not re-invent the wheel I think Jericho Source has a method for extracting just text.